src/notation/view/notationtoolbarmodel.cpp

```diff
@@ -89,9 +91,9 @@ void NotationToolBarModel::load()
     });
 }
 
-void NotationToolBarModel::handleAction(const QString& actionCode)
+void NotationToolBarModel::handleMenuItem(const QString& itemId)
 {
-    dispatcher()->dispatch(actions::codeFromQString(actionCode));
+    dispatcher()->dispatch(actions::codeFromQString(itemId));
```

Comment:
I believe you need to explicitly find the item by itemId and get actionCode from it